validator/sawtooth_validator/journal/chain.py

```diff
@@ -525,9 +556,27 @@ def on_block_validated(self, commit_new_block, result):
 
                 # if the head has changed, since we started the work.
                 if result["chain_head"] != self._chain_head:
-                    # chain has advanced since work started.
-                    # the block validation work we have done is saved.
-                    self._verify_block(new_block)
+                    LOGGER.info(
+                        'Chain head updated from %s to %s while processing '
+                        'block: %s',
+                        result["chain_head"].identifier[:8],
+                        self._chain_head.identifier[:8],
```

Comment:
Can do, although I did this because all I really cared about in the message was the identifier part.